web/src/connectors/categories-table-connector/categories-table-connector.tsx

```diff
@@ -46,7 +46,8 @@ export const CategoriesTableConnector: FC<CategoriesTableConnectorProps> = () =>
         },
         [data?.data]
     );
-
+    //TODO need to fix
+    //@ts-ignore
```

Comment:
can we try to get rid of ts-ignors?

Reply:
I suppose it would be better to remove all ts-ignore in project (and maybe some of them are connected, so that's why it need to be removed together)
<img width="292" alt="image" src="https://user-images.githubusercontent.com/50149163/212062153-11a8ffc8-380e-4525-a5f0-e586eaa63108.png">

Reply:
yes that would be great to get rid of all ts-ignors, but it should be done in a refactoring task. Here we are discussing this PR and it would be great if you will not add new ts-ignors to the project. If it's a difficult change then it could be done in a separate task.